src/main/java/org/company/demo/library/libraryproject/mappers/mapperinterface/BookMapperInterface.java

```diff
@@ -0,0 +1,13 @@
+package org.company.demo.library.libraryproject.mappers.mapperinterface;
+
+import org.company.demo.library.libraryproject.dto.BookDTO;
+import org.company.demo.library.libraryproject.models.Book;
+
+public interface BookMapperInterface {
```

Comment:
Usually the interface has the generic name as that is used as type name for the field.